docs/workflow/building/coreclr/README.md

```diff
@@ -1,100 +1,156 @@
 
-By default, the script generates a _Debug_ build type, which is not optimized code and includes asserts. As its name suggests, this makes it easier and friendlier to debug the code. If you want to make performance measurements, you ought to build the _Release_ version instead, which doesn't have any asserts and has all code optimizations enabled. Likewise, if you plan on running tests, the _Release_ configuration is more suitable since it's considerably faster than the _Debug_ one. For this, you add the flag `-configuration release` (or `-c release`). For example:
+- `corerun`: The command-line host executable. This program loads and starts the CoreCLR runtime and receives the managed program you want to run as argument (e.g. `./corerun program.dll`). On Windows, it is called `corerun.exe`.
+- `coreclr`: The CoreCLR runtime itself. On Windows, it's called `coreclr.dll`, on macOS it is `libcoreclr.dylib`, and on Linux it is `libcoreclr.so`.
+- `System.Private.CoreLib.dll`: The core managed library, containing the definitions of `Object` and the base functionality.
 
-```bash
-./build.sh --subset clr --configuration release
-```
+All the generated logs are placed in under `artifacts/log`, and all the intermediate output the build uses is placed in the `artifacts/obj/coreclr` directory.
 
-As mentioned before in the [general building document](/docs/workflow/README.md#configurations-and-subsets), CoreCLR also supports a _Checked_ build type which has asserts enabled like _Debug_, but is built with the native compiler optimizer enabled, so it runs much faster. This is the usual mode used for running tests in the CI system.
+### What to do with the Build
 
-Now, it is also possible to select a different configuration for each subset when building them together. The `--configuration` flag applies universally to all subsets, but it can be overridden with any one or more of the following ones:
+*CoreCLR* is one of the most important components of the runtime repo, as it is one of the main engines of the .NET product. That said, while you can test and use it on its own, it becomes easiest to do this when used in conjuction with the *Libraries* subset. When you build both subsets, you can get access to the *Core_Root*. This includes all the libraries, as well as the Clr alongside other tools like *Crossgen2*, *R2RDump*, and the *ILC* compiler, and the main command-line host executable `corerun`, all bundled together. The *Core_Root* is one of the most reliable ways of testing changes to the runtime, running external apps with your build, and it is the way Clr tests are run in the CI pipelines.
 
-* `--runtimeConfiguration (-rc)`: Flag for the CLR build configuration.
-* `--librariesConfiguration (-lc)`: Flag for the libraries build configuration.
-* `--hostConfiguration (-hc)`: Flag for the host build configuration.
+#### The Core Root for Testing Your Build
```

Comment:
I think it is important to mention the `CORE_ROOT` is for testing. Otherwise, it might imply it's another part of the product to newcomers and less experienced users.

Reply:
Can you put an underscore between Core and Root to keep it consistent?